src/platform/forms/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -142,6 +142,8 @@ class SaveInProgressIntro extends React.Component {
     } else if (renderSignInMessage) {
       alert = renderSignInMessage(prefillEnabled);
     } else if (prefillEnabled && !verifyRequiredPrefill) {
+      // If you pass in a headingLevel prop it will change what is rendered by the changeSaveInProgressHeadingLevel helper
```

Comment:
We don't need the reference to the `changeSaveInProgressHeadingLevel helper` anyore.

Reply:
Removed this